draft-ietf-quic-http.md

```diff
@@ -433,28 +433,43 @@ frame.
 
 ## Server Push
 
-HTTP/QUIC supports server push as described in {{!RFC7540}}. During connection
-establishment, the client enables server push by sending a MAX_PUSH_ID frame
-(see {{frame-max-push-id}}).  A server cannot use server push until it receives
-a MAX_PUSH_ID frame.
+HTTP/QUIC supports server push in a similar manner to {{!RFC7540}}, but uses
+different mechanisms. During connection establishment, the client enables server
+push by sending a MAX_PUSH_ID frame (see {{frame-max-push-id}}). A server cannot
+use server push until it receives a MAX_PUSH_ID frame.
```

Comment:
Which makes me wonder (probably for the second time) whether adding a setting is valuable or not.  I guess not because MAX_PUSH_ID can be send immediately after SETTINGS, and having two ways to do the same thing isn't great.

Reply:
By that logic, we shouldn't have transport parameters for max stream ID either -- just send MAX_STREAM_ID frames in your first packet.  But that would prevent servers from using 0.5-RTT data, since they haven't received authenticated packets from the client yet.

HTTP has no such restriction, since all data is inside the crypto.  We can just use the frames.